### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by dan-abramov**

### CODE REVIEW

File: app/policies/admin_policy.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AdminPolicy < ApplicationPolicy
+  def has_rights?
+    return true if @user.has_role? :admin
```
Comment (X1ting) [inline app/policies/admin_policy.rb]: are you really need `return true` ?

File: db/seeds.rb [hunk 2]
```diff
@@ -8,13 +8,16 @@
...
+#   Card.create(original_text: original, translated_text: translated, user_id: 17)
+# end
 
-doc.search('//table/tbody/tr').each do |row|
-  original = row.search('td[2]/p')[0].content.downcase
-  translated = row.search('td[1]/p')[0].content.downcase
-  Card.create(original_text: original, translated_text: translated, user_id: 17)
-end
\ No newline at end of file
+@user = User.create(email:'admin@mail.com', password:'12345678', password_confirmation:'12345678')
```
Comment (X1ting) [inline db/seeds.rb]: why you need instance variable?

File: app/admin/dashboard.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
+ActiveAdmin.register_page "Dashboard" do
+  controller do
+    before_action :check_authorization
+
+    private
+
+    def check_authorization
+
```
Comment (X1ting) [inline app/admin/dashboard.rb]: extra new-line

**Review by X1ting — CHANGES_REQUESTED** [review]

File: db/seeds.rb (hunk 2 above)
Comment (dan-abramov) [reply db/seeds.rb]: I don't need this, actually. It created, because I used to do it in controllers :)

**Review by X1ting — APPROVED** [review]

**Pull request merged**